src/components/pages/pitch-arena/PitchPage.tsx

```
@@ -0,0 +1,316 @@
+  <button
+    onClick={() => (window.location.href = "/booking")}
+    style={{
+      padding: '16px 48px',
+      backgroundColor: '#dc2626',
+      color: 'white',
+      borderRadius: '9999px',
+      fontSize: '18px'
+    }}
+    className="shadow-lg hover:scale-105 transition"
+  >
+    Register Now
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)